remoteit/desktop
# [DESK-434] Move network and hosted tabs to settings page.

### Description

Moved "Hosted" and "Network" pages under settings.
See ticket for details and wireframes.

### Checklist




### Test plan



1. Sign in
2. See that _Hosted_ and _Network_ tabs are gone
3. Go to _Settings_ page
4. If unregistered see blue text at top indicating how to register
5. Click the register link
6. Register your device
7. Scan for servies
8. Add service
9. Return to settings page
10. Settings page should show system name

### Ticket



<https://remoteit.atlassian.net/browse/DESK-434>
